File: Modul-2/soal-4/PRAK204_2410817210016_MUHAMMADRIZKYAKBAR.c
#include <stdio.h>
#define pi 22.0/7.0

int main ()
{
    double jari, tinggi;
    double kel, vol, luas;

    printf("Masukkan Jari Jari : ");
    scanf("%lf", &jari);
    printf("Masukkan Tinggi Bejana : ");
    scanf("%lf", &tinggi);

    vol = pi * jari * jari * tinggi;
    luas = 2 * pi  * jari * (jari + tinggi);
    kel = 2 * pi * jari;

    printf("Volume = %.2lf\n", vol);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", kel);
    return 0;
}

File: Modul-2/soal-4/PRAK204_2410817210016_MUHAMMADRIZKYAKBAR.py
r = float(input("Masukkan jari-jari: "))
t = float(input("Masukkan tinggi: "))

pi = 22 / 7

volume = pi * r**2 * t

luas = 2 * pi * r * (r + t)

keliling = 2 * pi * r

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")
